### ISSUE TITLE
Error when running init mix task in an alias

### ISSUE THREAD

**Issue by zamith** [commanded/eventstore#160]
We have an alias that runs `event_store.init` followed by `ecto.migrate`. Since upgrading to Ecto 3.0 we've gotten the following issue:

```
The EventStore database has been initialized.
[info] == Running 20180212111921 Migration.change/0 forward
...
[info] == Migrated 20180827100604 in 0.0s
** (EXIT from #PID<0.91.0>) killed
```

Without the `event_store.init` it runs fine. I would guess that there is some mismanagement of connections somewhere, but can't pinpoint where. Any idea of what's going on?

To add some more context, if run separately it also works fine. The issue only occurs when running them consecutively in a mix alias.

**Comment by slashdotdash** [comment]
Yes, this issue has been [fixed in v0.16.1](https://github.com/commanded/eventstore/blob/master/CHANGELOG.md#v0161).

**Comment by zamith** [comment]
Thanks. It works.
